.husky/pre-commit

```diff
@@ -0,0 +1 @@
+yarn run pre-commit
```

Comment:
This is the configuration style Husky has used since v5, so this file replaces the `hooks.pre-commit` property in `package.json`